internal/Picker/Picker.js

```diff
@@ -203,7 +203,24 @@ const PickerBase = kind({
 		 * @default 0
 		 * @public
 		 */
-		value: PropTypes.number
+		value: PropTypes.number,
+
+		/**
+		 * Choose a specific size for your picker. `'small'`, `'medium'`, `'large'`, or set to `null` to
+		 * assume auto-sizing. `'small'` is good for numeric pickers, `'medium'` for single or short
+		 * word pickers, `'large'` for maximum-sized pickers.
+		 *
+		 * You may also supply a number. This number will determine the minimum size of the Picker.
+		 * Setting a number to less than the number of characters in your longest value may produce
+		 * unexpected results.
+		 *
+		 * @type {('small'|'medium'|'large'|Number)}
+		 * @public
+		 */
+		width: PropTypes.oneOfType([
+			PropTypes.oneOf([null, 'small', 'medium', 'large']),
```

Comment:
Are there any code to handle `'small', 'medium', 'large'` values in Agate Pickers like Sandstone? If not, let's support numbers. Then we might fix the qa-a11y Picker view using width prop of `large` and `medium`.

```
	determineClasses (css, decrementerDisabled, incrementerDisabled) {
		...

		return classnames(
			css.picker,
			css[orientation],
			css[width],
```

Reply:
We will support those values once animation related code got in soon.